app/api/constituency_lookup/[postcode]/[[...address]]/route.ts

```diff
@@ -18,27 +18,25 @@ console.log(`***** dbPath is [${dbPath}] *****`);
 // TODO: Next.js doesn't cache POSTs - potentially change to a GET of format:
 // constituency_lookup/{postcode}/{addressSlug}
 // which should enable caching
-export async function POST(request: NextRequest) {
+export async function GET(request: NextRequest, context) {
```

Comment:
It seems like if you use the `request` param, the route will be opted-out of caching :thinking: 
https://nextjs.org/docs/app/building-your-application/routing/route-handlers#caching

The dynamic route segments section doesn't say anything about caching:
https://nextjs.org/docs/app/building-your-application/routing/route-handlers#caching

It may be that if we have `request` in the function signature, but don't access it, caching will still work, but not sure.
We're not actively using it anymore tho, so we can remove the `console.log(request)` a few lines down - I suspect having that will _definitely_ stop caching, but maybe the dynamic routes also prevents caching :roll_eyes: